Assets/VRTK/Scripts/Locomotion/VRTK_GazeDrivenLocomotion.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Gaze Driven Movement|Scripts|0070
+namespace VRTK
+{
+    using UnityEngine;
+    using UnityEngine.UI;
+    /// <summary>
+    /// The Gaze Driven Movement script creations a directional "joystick" underneath a given game object within the seen. Movement is driven by moving the joystick using the HMD to drag across the gamepad.
+    /// </summary>
+    /// <remarks>
+    /// This script is meant to be used for 3rd person VR games.
+    /// Attach this script to the `[CameraRig]` prefab. Attach the VRTK_DirectionPad prefab to get the UI for the directional pad.
+    /// </remarks>
+    public class VRTK_GazeDrivenLocomotion: MonoBehaviour
+    {
+        [Tooltip("The speed of the character when HMD gaze is at the edge of the game pad")]
+        [SerializeField]
+        private float maxSpeed = 2.0f;
```

Comment:
Why are these set to private and Serialized? do you not want users to be able to change these values at runtime?